pkg/api/v1/schema.go

```diff
@@ -794,6 +794,16 @@ type Interface struct {
 	// If specified, the virtual network interface will be placed on the guests pci address with the specifed PCI address. For example: 0000:81:01.10
 	// +optional
 	PciAddress string `json:"pciAddress,omitempty"`
+	// If specified the network interface will pass additional DHCP options to the VMI
+	// +optional
+	DhcpOptions `json:"dhcpOptions,omitempty"`
```

Comment:
The direction looks good. I would expect this option in `PodNetwork` in `spec.networks`, since we can only guarantee it there. Only there we know for sure that we run a DHCP server. What do you think?

Reply:
From what I can see, we run a DHCP server for every interface.

Reply:
Not always (if the interface in the pod has no IP, we just pass through without starting our own dhcp server), but good point. In most cases we do. Especially if they are normal k8s-network extensions.